kubectl-plugin/kubectl-waiops

```diff
@@ -1236,10 +1232,11 @@ then
                     getSubscriptionStatusGrep "ibm-automation-flink" "openshift-operators" 
                 fi
                 getSubscriptionStatusGrep "aiops-ibm-common-services" "openshift-operators"
-            else
-                echo "______________________________________________________________"
-                echo "Subscriptions from openshift-operators namespace:" && echo ""
-                getSubscriptionStatus "ibm-cert-mgr" "openshift-operators"
+            # else
+            # TODO: update this to check for cert manager certificate expiry
+            #     echo "______________________________________________________________"
+            #     echo "Subscriptions from openshift-operators namespace:" && echo ""
+            #     getSubscriptionStatus "ibm-cert-mgr" "openshift-operators"
```

Comment:
If this code is not needed, can we go ahead and delete this?

Reply:
I figured we should keep this commented out for now because we will address this is another open issue -- so we can keep this here to signify where the `ibm-cert-mgr` cert checks will be placed.

Reply:
But for now, I'll remove the commented out code and keep the TODO so we know where the cert checks should live.